terraform/gcp/projects/pangeo-hubs.tfvars

```diff
@@ -1,45 +1,20 @@
 prefix                 = "pangeo-hubs"
 project_id             = "pangeo-integration-te-3eea"
-enable_private_cluster = true
-
 core_node_machine_type = "n1-highmem-4"
+enable_private_cluster = true
 
 # Multi-tenant cluster, network policy is required to enforce separation between hubs
-enable_network_policy    = true
+enable_network_policy  = true
 
 # Some hubs want a storage bucket, so we need to have config connector enabled
 config_connector_enabled = true
 
+# Setup a filestore for in-cluster NFS
+enable_filestore = true
 
-notebook_nodes = {
-  "small" : {
-    min : 0,
-    max : 100,
-    machine_type : "n1-standard-2",
-    labels: {}
-  },
-  "medium" : {
-    min : 0,
-    max : 100,
-    machine_type : "n1-standard-8",
-    labels: {}
-  },
-  "large" : {
-    min : 0,
-    max : 100,
-    machine_type : "n1-standard-16",
-    labels: {}
-  },
-  "very-large" : {
-    min : 0,
-    max : 100,
-    machine_type : "n1-standard-32",
-    labels: {}
-  },
-}
```

Comment:
I guess there is a need to modify the profile in the pangeo config file, right?
IIRC, it is referencing these specific nodes...

Reply:
Correct. I have been working on this in a different branch (linked in top comment of #666) and will edit and open a PR ASAP on Monday.

Reply:
PR for profile list https://github.com/2i2c-org/pilot-hubs/pull/674